android/app/src/test/java/com/getcapacitor/myapp/ExampleUnitTest.java

```diff
@@ -0,0 +1,18 @@
+package com.getcapacitor.myapp;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+
+    @Test
+    public void addition_isCorrect() throws Exception {
+        assertEquals(4, 2 + 2);
```

Comment:
good that we have some test coverage here :trollface:

Reply:
I would still rather keep this, just in case there is no confusion if we need to generate android/ios platform from scratch, and wonder where did this file came from disappear. But if you are strongly against it, I can delete it as well.